WebDevStudios/wd_s
# Feature/#495 gutenberg block previews

Closes #495

### DESCRIPTION ###
- Adds previews to our custom blocks for WP 5.3.

### SCREENSHOTS ###
![](https://dl.dropbox.com/s/t36c09p4x9z3vrm/wp53-block-preview.gif?dl=0)

### OTHER ###

### STEPS TO VERIFY ###
Use [WordPress Beta Tester](https://wordpress.org/plugins/wordpress-beta-tester/) to install the latest nightly version of WP. When active, you'll see what is in the GIF above where blocks now have mini previews. There aren't previews for Recent or Related posts as these are dynamic; you'll notice this is the same outcome for various dynamic core blocks as well so I opted to run with that.

This does nothing in the current version of WordPress, so could likely be safely merged in and have no effects at all. In my testing down to 5.2.4, the code remains in place and has no positive or negative affect on the blocks.

### DOCUMENTATION ###
Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

Yep! If approved and merged in, I'll add notes on how to add the example/preview data for blocks.
